.github/workflows/dependency-scan.yml

```diff
@@ -4,16 +4,10 @@ on: push
 jobs:
   analyze:
     runs-on: ubuntu-latest
+    env:
+      FOSSA_API_KEY: ${{ secrets.FOSSA_API_KEY }}
     steps:
       - uses: actions/checkout@v4
       - run: curl https://raw.githubusercontent.com/fossas/fossa-cli/master/install-latest.sh | bash
-
-      # Just try to run fossa up to twice, for the case of temporary network or service issues.
-      - run: fossa analyze || fossa analyze
```

Comment:
Guessing this hasn't been much of an issue to warrant keeping this? Seems pretty hacky anyways.

Reply:
Yeah I don't think it ever actually was an issue, I think we did it "just in case"